oracle-linux-image-tools/cloud/vagrant-virtualbox/provision.sh

```diff
@@ -0,0 +1,196 @@
+  /usr/sbin/useradd vagrant -g vagrant -G wheel
+  echo "%vagrant ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers.d/vagrant
+  chmod 0440 /etc/sudoers.d/vagrant
+  restorecon /etc/sudoers.d/vagrant
+  sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers
+
+  # sshd: disable password authentication and DNS checks
+  ex -s /etc/ssh/sshd_config <<EOF
+:%substitute/^\(PasswordAuthentication\) yes$/\1 no/
+:%substitute/^#\(UseDNS\) yes$/&\r\1 no/
+:update
+:quit
+EOF
+
+  cat >>/etc/sysconfig/sshd <<EOF
+
+# Decrease connection time by preventing reverse DNS lookups
+# (see https://lists.centos.org/pipermail/centos-devel/2016-July/014981.html
+#  and man sshd for more information)
+OPTIONS="-u0"
+EOF
+
+  # Default insecure vagrant key
+  mkdir -m 0700 -p /home/vagrant/.ssh
+  echo "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key" >> /home/vagrant/.ssh/authorized_keys
+  chmod 600 /home/vagrant/.ssh/authorized_keys
+  chown -R vagrant:vagrant /home/vagrant/.ssh
+
+  # Fix for issue #76, regular users can gain admin privileges via su
+  ex -s /etc/pam.d/su <<'EOF'
+/^account\s\+sufficient\s\+pam_succeed_if.so uid = 0 use_uid quiet$/
+:append
+# allow vagrant to use su, but prevent others from becoming root or vagrant
+account         [success=1 default=ignore] \\
+                                pam_succeed_if.so user = vagrant use_uid quiet
+account         required        pam_succeed_if.so user notin root:vagrant
+.
+:update
+:quit
+EOF
+
+  echo 'vag' > /etc/yum/vars/infra
+
+  # Configure grub to wait just 1 second before booting
+  sed -i 's/^GRUB_TIMEOUT=[0-9]\+$/GRUB_TIMEOUT=1/' /etc/default/grub
+  grub2-mkconfig -o /boot/grub2/grub.cfg
+
+  # Blacklist the floppy module to avoid probing timeouts
+  echo blacklist floppy > /etc/modprobe.d/nofloppy.conf
+  chcon -u system_u -r object_r -t modules_conf_t /etc/modprobe.d/nofloppy.conf
+
+  # Customize the initramfs
+  # Enable VMware PVSCSI support for VMware Fusion guests.
+  echo 'add_drivers+=" mptspi "' > /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  restorecon /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  # There's no floppy controller, but probing for it generates timeouts
+  echo 'omit_drivers+=" floppy "' > /etc/dracut.conf.d/nofloppy.conf
+  restorecon /etc/dracut.conf.d/nofloppy.conf
+  # Regenerate initrd
+  local current_kernel=$(uname -r)
+  ${DRACUT_CMD} -f "/boot/initramfs-${current_kernel}.img" "${current_kernel}"
+
+  # Set SELinux to enforcing
+  sed -i -e 's/^SELINUX\s*=.*/SELINUX=enforcing/' /etc/selinux/config
+
+  # Install additional Packages
+  yum install -y ${YUM_VERBOSE} wget \
+    oracle-epel-release-el7 \
```

Comment:
What's the rationale behind enabling repos that are unsupported? I think the base boxes should not have any EPEL or developer repos enabled because that would immediately make the box unsupportable.

Reply:
For compatibility with the current boxes.
Made this optional in 21e0369